### ISSUE TITLE
Wrong values for Ebit and missing values for Extraordinary items

### ISSUE THREAD

**Issue by NicsunXnus** [ranaroussi/yfinance#1044]
1) I have noticed that the values for Ebit are the same as Operating income for some stocks. I have tried this with 2 stocks "GOOG" and "TSM" and they both showed Ebit values that are the same as Operating income. However this is not the case for "D05.SI" and instead the Operating Income is the same as the Income before Tax. I was unable to verify for this stock since much of the data is not found directly on yahoo finance so I assume that they must have been calculated within the code.

2) Extraordinary items are showed as none for some stocks, even when they are displayed on the yahoo finance page. Example stocks used are "D05.SI","GOOG" & "TSM". I am thinking this is because of naming issues, as on the yahoo finance page, it is listed as unusual items.

Thank you

**Comment by ValueRaider** [comment]
Did you search through existing issues before submitting? Your issue is familiar.

**Comment by NicsunXnus** [comment]
Maybe I was a bit too hasty when I posted :P. I just searched through the issues and under #1004, found that there may be a possible fix from git-shogg's code (for the ebit and operating income identical values issue)? I notice that there is a pending merge request of his code into this codebase so hopefully that gets settled soon!

Regarding the missing extraordinary items issue, I am certain that it is a new issue as I was not able to find any similar issues under the open and closed issues sections.
